front/pages/api/w/[wId]/labs/transcripts.ts

```diff
@@ -0,0 +1,153 @@
+import type { WithAPIErrorReponse } from "@dust-tt/types";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { Authenticator, getSession } from "@app/lib/auth";
+import type { LabsTranscriptsProviderType } from "@app/lib/labs/transcripts/utils/types";
+import { LabsTranscriptsConfigurationResource } from "@app/lib/resources/labs_transcripts_configuration_resource";
+import { apiError, withLogging } from "@app/logger/withlogging";
+import { launchRetrieveNewTranscriptsWorkflow } from "@app/temporal/labs/client";
+
+export type GetLabsTranscriptsConfigurationResponseBody = {
+  configuration: LabsTranscriptsConfigurationResource | null;
+};
+
+async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<
+    WithAPIErrorReponse<GetLabsTranscriptsConfigurationResponseBody>
+  >
+): Promise<void> {
+  const session = await getSession(req, res);
+  const auth = await Authenticator.fromSession(
+    session,
+    req.query.wId as string
+  );
+
+  const owner = auth.workspace();
+  if (!owner) {
```

Comment:
You can gate behind `auth.isBuilder` and ensure that workspace has the FF (Feature Flag) enabled.

Reply:
Actually after discussion, we will let all users access the feature so only FF check